index.html

```diff
@@ -0,0 +1,321 @@
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <section class="contact">
+        <div>
+          <h2 class="contact-heading">Contact me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it.</p>
+        </div>
+        <form class="form">
+          <div>
+            <input type="text" id="name" name="name" placeholder="your name" required>
+          </div>
+          <div>
+            <input type="email" id="email" name="email" placeholder="your email" required>
+          </div>
+          <div>
+            <textarea name="message" id="message" placeholder="tell me something mate!" cols="30" rows="10" required></textarea>
```

Comment:
- [ ] Kindly use the property `maxlength="500"` attribute. With this attribute, you will specify the maximum length of the input string in characters. so the browser is expected to enforce this, by refusing to accept more than 500 characters. Please add this attribute🛠️ 🥇